client.go

```diff
@@ -36,22 +36,22 @@ import (
 	"github.com/pilosa/pilosa/internal"
 )
 
-// ClientOptions represents the configuration for a Client
+// ClientOptions represents the configuration for a InternalHTTPClient
 type ClientOptions struct {
 	TLS *tls.Config
 }
 
-// Client represents a client to the Pilosa cluster.
-type Client struct {
-	host    *URI
-	options *ClientOptions
+// InternalHTTPClient represents a client to the Pilosa cluster.
+type InternalHTTPClient struct {
+	defaultURI *URI
+	options    *ClientOptions
 
 	// The client to use for HTTP communication.
 	HTTPClient *http.Client
 }
 
-// NewClient returns a new instance of Client to connect to host.
-func NewClient(host string, options *ClientOptions) (*Client, error) {
+// NewClient returns a new instance of InternalHTTPClient to connect to host.
+func NewClient(host string, options *ClientOptions) (*InternalHTTPClient, error) {
```

Comment:
Any reason not to call this `NewInternalHTTPClient()`?

Reply:
I forgot to rename that function, going to do.